tuples.md

```diff
@@ -0,0 +1,67 @@
+# Writing Assignment - Tuples
+
+So far you have seen how to use arrays for holding a set of values and then iterate through them, for example:
+
+```swift
+var distances: [Int] = [10, 15, 20]
+
+for distance in distances {
+  print(distance)
+}
+```
+
+Arrays are a useful method for collecting variables, but limited as the values they contain can only be of one type. In this exercise we will introduce another, more flexible content type, the tuple.
+
+A tuple is a flexible variable type that allows you to group multiple variables into one convenient compound variable. The variables contained within a tuple can be of any type (for example a string, integer or even arrays) and you can mix the types. This makes tuples useful for creating a container variable for related values, for example:
+
+```swift
+let completedDistance = ("km", 10, true)
+```
+
+This tuple represents an entry from a fitness tracker, recording a distance unit as a string, a distance as an integer and a boolean to show completion.
+
+**Note**: Tuples are designed for short term groups of related data and not for data that is accessed frequently in an application. Long term data storage is generally handled by classes, which you will learn later.
+
```

Comment:
While this is true, we probably don't need this here especially since students don't know yet what classes are. It doesn't really add to their knowledge.